web/controllers/v1/group_membership_controller.ex

```diff
@@ -18,6 +18,13 @@ defmodule Cog.V1.GroupMembershipController do
     render(conn, "show.json", group: group)
   end
 
+  def manage_group_users(conn, %{"users" => user_spec}=params) do
+    params
+    |> Map.put("members", %{"users" => user_spec})
+    |> Map.delete("users")
+    |> fn(param) -> manage_membership(conn, param) end.()
```

Comment:
This reads really weirdly. Any reason to not just edit `manage_membership/2` to get rid of the roles and groups instead?